web/packages/build/jest/config.js

```diff
@@ -31,6 +31,7 @@ module.exports = {
     '^e-teleport/(.*)$': '<rootDir>/e/web/teleport/src/$1',
     '^gen-proto-js/(.*)$': '<rootDir>/gen/proto/js/$1',
     '^gen-proto-ts/(.*)$': '<rootDir>/gen/proto/ts/$1',
+    '^tmp/(.*)$': '<rootDir>/tmp/$1',
```

Comment:
As I mentioned on Slack, I'd consider dropping this since if it's just a single file that needs it, it can be replaced with a relative import, something like `../../tmp/preset-roles.json`.

Reply:
ok